### DISCUSSION TITLE
How do I get Bootstrap to work on Email Template with Laravel Email Verification Templates?

### DISCUSSION THREAD

**Discussion by nanakumi75** [twbs/bootstrap#40930] [Q&A] [answered]
I am using Bootstrap for interface design on a Laravel Project. I need to design an Email Verification Template for users to activate their account during registration.
This template is sent to the user's email with a link to click to activate their account. I realize that Bootstrap Classes don't work in the email template sent to the user's email after registration on my website. 
Am I doing something wrong or Bootstrap is not supposed to work on the backend?

**Comment by julien-deramond** [comment, marked as answer]
> I realize that Bootstrap Classes don't work in the email template sent to the user's email after registration on my website.

Bootstrap wasn't designed for email templates, so I don't know exactly the limitations and how you should adapt it.
The answer would depend on the classes that don't work. All classes, only a few, which ones?

Anyway, most email clients have poor or inconsistent support for external styles, JavaScript, and advanced CSS features. Even some basic CSS rules may not work reliably across all email platforms. So, email templates often require styles to be applied inline.
That's one solution.

I checked very quickly out there, and I stumbled on https://bootstrapemail.com/. I never tried it, but this kind of tool/library could be a solution to your problem too, or even tools no relying on Bootstrap :)

**Comment by nanakumi75** [comment]
Thanks for the message. I will stick to designing the template using a native css. It works fine
